templates/base.html.twig

```diff
@@ -10,88 +10,87 @@
+                            <span class="ms-1">Users</span>
                           </a>
-                        </li>
-                        <li class="nav-item">
-                          <a class="nav-link{% if menuactive == "logs" %} active{% endif %}" href="{{ path('app_logs') }}">
+                          <a title="Logs" class="text-nowrap nav-link{% if menuactive == "logs" %} active{% endif %}" href="{{ path('app_logs') }}">
                             <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6">
                               <path stroke-linecap="round" stroke-linejoin="round" d="M19.5 14.25v-2.625a3.375 3.375 0 00-3.375-3.375h-1.5A1.125 1.125 0 0113.5 7.125v-1.5a3.375 3.375 0 00-3.375-3.375H8.25m0 12.75h7.5m-7.5 3H12M10.5 2.25H5.625c-.621 0-1.125.504-1.125 1.125v17.25c0 .621.504 1.125 1.125 1.125h12.75c.621 0 1.125-.504 1.125-1.125V11.25a9 9 0 00-9-9z" />
                             </svg>
-                            <span class="ms-2">Logs</span>
+                            <span class="ms-1">Logs</span>
                           </a>
-                        </li>
+
+                          <a title="Check" class="text-nowrap nav-link nav-link-extra link-secondary" href="{{ path('app_checkmailnow') }}">
+                            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-check-circle" viewBox="0 0 16 16">
+                              <path d="M8 15A7 7 0 1 1 8 1a7 7 0 0 1 0 14m0 1A8 8 0 1 0 8 0a8 8 0 0 0 0 16"/>
+                              <path d="m10.97 4.97-.02.022-3.473 4.425-2.093-2.094a.75.75 0 0 0-1.06 1.06L6.97 11.03a.75.75 0 0 0 1.079-.02l3.992-4.99a.75.75 0 0 0-1.071-1.05"/>
+                            </svg>
+                            <span class="ms-1">Check</span>
+                          </a>
+                          <a title="Logs" class="text-nowrap nav-link nav-link-extra link-danger" href="{{ path('app_logout') }}">
+                            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-door-open" viewBox="0 0 16 16">
+                              <path d="M8.5 10c-.276 0-.5-.448-.5-1s.224-1 .5-1 .5.448.5 1-.224 1-.5 1"/>
+                              <path d="M10.828.122A.5.5 0 0 1 11 .5V1h.5A1.5 1.5 0 0 1 13 2.5V15h1.5a.5.5 0 0 1 0 1h-13a.5.5 0 0 1 0-1H3V1.5a.5.5 0 0 1 .43-.495l7-1a.5.5 0 0 1 .398.117M11.5 2H11v13h1V2.5a.5.5 0 0 0-.5-.5M4 1.934V15h6V1.077z"/>
+                            </svg>
+                            <span class="ms-1">Logout</span>
+                          </a>
+
+                          <a class="nav-link"></a>
                         {% endif %}
-                      </ul>
-                </div>
-            </nav>
+              </nav>
             {% endif %}
-            <main class="col-md-9 ms-sm-auto col-lg-11 px-md-4 py-4">
```

Comment:
please revert.

Reply:
Revert what exactly?

Reply:
you basicly removed all the spacing making the entire design cramped and text heavy.

Reply:
I didn't. I just moved it. The <main> tag still exists.

Reply:
what im referring to is 
![image](https://github.com/antedebaas/DMARC-SMTPTLS-Reports/assets/5467398/d819a330-2a71-4854-b16b-1b8b8430e0cd)
that i want to keep this spacing

Reply:
OK for >=xl (which was the only case before I started).